src/repl/scala/tools/nsc/interpreter/ReplProps.scala

```diff
@@ -3,26 +3,27 @@
  * @author Paul Phillips
  */
 
-package scala.tools.nsc
-package interpreter
+package scala.tools.nsc.interpreter
 
-import Properties.{ javaVersion, javaVmName, shellPromptString, shellWelcomeString,
-                    versionString, versionNumberString }
+import java.io.File
+import java.util.{Formattable, FormattableFlags, Formatter}
+
+import scala.tools.nsc.Properties
+import Properties._
 import scala.sys.{BooleanProp, Prop}
 import scala.sys.Prop._
-import java.util.{ Formattable, FormattableFlags, Formatter }
 
-class ReplProps {
+trait ReplProps {
   private def bool(name: String) = BooleanProp.keyExists(name)
   private def int(name: String)  = Prop[Int](name)
 
   // This property is used in TypeDebugging. Let's recycle it.
-  val colorOk = Properties.coloredOutputEnabled
+  private val colorOk = Properties.coloredOutputEnabled
```

Comment:
It would be so nice if `private` had `private[this]` semantics. I don't suppose you could sneak that in under cover of the construction dust.